### ISSUE TITLE
Use case-sensitive collation for benchmarkId in MySQL

### ISSUE THREAD

**Issue by csmig** [NUWCDIVNPT/stig-manager#205]
**TLDR**
Change the collation of all benchmarkId columns from the MySQL default of `utf8mb4_0900_as_ci` to `utf8mb4_0900_as_cs` 

**Justification**
DISA regularly publishes new STIG releases whose primary key (benchmarkId) differs from the previous release. The latest occurrence in January 2021 was a primary key change from `Mozilla_FireFox_STIG` to `Mozilla_Firefox_STIG`. This astonishing practice creates challenges for STIG Manager because JavaScript property names are case-sensitive but the default MySQL collation is not.

Unlike Oracle (the backend for STIGMAN classic), MySQL does **not** automatically use a case-sensitive collation for primary keys. So we should change the collation manually.
